cluster-provision/okd/scripts/provision.sh

```diff
@@ -302,13 +281,47 @@ function get_value()
     echo "$val"
 }
 
-# Remove master schedulable taint from masters
+# in case of one master-node: 
+# Disable updates of machines configurations, because on the update the machine-config
+# controller will try to drain the master node, but it not possible with only one master
+# so the node will stay in cordon state forewer.
+# It will prevent any updates of the kubelet or registries configuration on the machine
+# so if you need some, please add it before these lines
+
+#uncomment the next lines in case of one master
+
+
 masters=$(get_value 50 oc get nodes -l node-role.kubernetes.io/master -o'custom-columns=name:metadata.name' --no-headers)
+
 if [[ ${masters} == "" ]]; then
     echo "Failed to get masters list, exiting"
     exit 1
 fi
 
+if [[ "$(echo ${masters[@]} | wc -w)" -le "1" ]]; then
```

Comment:
Awesome,
imo i would use this var
https://github.com/kubevirt/kubevirtci/blob/0c328ba07125320f429d136f3ce1afe20cae32b0/cluster-provision/okd/scripts/provision.sh#L16

Reply:
DONE